src/providers/image-loader.ts

```diff
@@ -67,11 +69,28 @@ export class ImageLoader {
   }
 
   private get isWKWebView(): boolean {
-    return this.platform.is('ios') && (<any>window).webkit;
+    if (this.platform.is('ios') && (<any>window).webkit && (<any>window).webkit.messageHandlers) {
```

Comment:
I don't know whether it is necessary or not. When I was inspecting the issue with Livereload, I saw people are using with **webkit.messageHandlers**.
https://stackoverflow.com/questions/28795476/detect-if-page-is-loaded-inside-wkwebview-in-javascript
Therefore, I made this change.